.github/workflows/build-scripts.yml

```diff
@@ -19,25 +19,8 @@ jobs:
       with:
         m2-cache-key: 'build-scripts'
 
-    - name: Run metabuild-common build script tests
-      run: clojure -M:test
-      working-directory: bin/common
-      timeout-minutes: 15
-    - name: Run build-drivers build script tests
-      run: clojure -M:test
-      working-directory: bin/build-drivers
-      timeout-minutes: 15
-    - name: Run i18n script tests
-      run: clojure -M:test
-      working-directory: bin/i18n
-      timeout-minutes: 15
-    - name: Run build-mb build script tests
-      run: clojure -M:test
-      working-directory: bin/build-mb
-      timeout-minutes: 15
-    - name: Run release script tests
-      run: clojure -M:test
-      working-directory: bin/release
+    - name: Run build and release script tests
+      run: clojure -X:dev:drivers:build:build-dev:build-test:ci
```

Comment:
I should also mention that we can run all the build/release script tests in one step instead of 5 separate steps now. Well we could have done this all along probably but it's much easier now. So this will be nice and speed up CI a bit. Not like this was super slow in the first place, but this will be a nice benefit

Reply:
For practical purposes in the CI, it's easier to see the log (and search for errors etc) on GitHub Actions UI when there are X separate steps. Otherwise, everything is drawn in the log of one very big (and log) step.

Reply:
Sure, but it probably takes an extra 10 seconds to spin up Clojure in each step and load all the same namespaces up since they're all basically different entry points to the same project, so shaving a minute off of the CI job is worth it I think. Also people almost never touch this stuff so these tests almost never fail